### PR TITLE
Remove Enum34 Dependency

### PR DESCRIPTION
**Pull request by piercefreeman**
Fix breakages in other places in the ecosystem.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,9 +19,10 @@
     keywords="microcosm",
     install_requires=[
         "click>=6.7",
-        "enum34>=1.1.6",
         "PyYAML>=3.12",
         "requests>=2.18.4",
+        # 09-21-2020: pin for enforce_white_space issue @pierce
+        "isort<5",
```
Comment (adamhadani) [inline setup.py:25]: @piercefreeman I've seen us pin this in our entrypoint.sh, do we actually want a version pin for a linter to be part of the `install_requires` of the package? cc/ @GabeLambda any input whether we should capture it here or elsewhere?

**Review by adamhadani — APPROVED** [review]
LGTM with one caveat re pinning version of `isort<5` as part of main setup.py `install_requires`

File: setup.py (hunk 1 above)
Comment (GabeLambda) [reply setup.py:25]: This belongs more in the `lint` section of `entrypoint.sh`. This isn't a direct dependency of the project.

File: entrypoint.sh [hunk 2]
```diff
@@ -20,20 +19,20 @@
...
+   pip --quiet install \
+       .[test] nose "PyHamcrest<1.10.0" coverage
+   exec nosetests
 elif [ "$1" = "lint" ]; then
-    # Install standard linting dependencies; YMMV
-    pip --quiet install \
-        .[lint] flake8 flake8-print flake8-logging-format
-    exec flake8 ${NAME}
+   # Install standard linting dependencies; YMMV
+   pip --quiet install \
```
Comment (adamhadani) [inline entrypoint.sh:28]: cc/ @piercefreeman this is the line where you'll want to pin isort e.g. add a `isort<5.0.0` pin to this pip install

**Pull request merged**